pkg/scheduler/framework/plugins/examples/multipoint/multipoint.go

```diff
@@ -27,6 +27,7 @@ import (
 type CommunicatingPlugin struct{}
 
 var _ = framework.ReservePlugin(CommunicatingPlugin{})
+var _ = framework.PrebindPlugin(CommunicatingPlugin{})
```

Comment:
Thanks for the update. Now that you are at it, could you please fix the `New` function to take `FrameworkHandle` instead of `Framework`?

Reply:
sure, done!